ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/TaskList/Decision/WhyDeferredIntegrationTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using ApplyToBecome.Data.Models;
+using ApplyToBecome.Data.Models.AdvisoryBoardDecision;
+using ApplyToBecomeInternal.Tests.PageObjects;
+using FluentAssertions;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ApplyToBecomeInternal.Tests.Pages.TaskList.Decision
+{
+	public class WhyDeferredIntegrationTests : BaseIntegrationTests, IAsyncLifetime
+	{
+		private AcademyConversionProject _project;
+		private RecordDecisionWizard _wizard;
+		private string PageHeading => Document.QuerySelector("h1").TextContent.Trim();
+		private IElement ErrorSummary => Document.QuerySelector(".govuk-error-summary");
+
+		public WhyDeferredIntegrationTests(IntegrationTestingWebApplicationFactory factory) : base(factory)
+		{
+		}
+
+		public Task DisposeAsync()
+		{
+			return Task.CompletedTask;
+		}
+
+		public async Task InitializeAsync()
+		{
+			_project = AddGetProject(p => p.GeneralInformationSectionComplete = false);
+			_wizard = new RecordDecisionWizard(Context);
+
+			await _wizard.StartFor(_project.Id);
+			await _wizard.SetDecisionToAndContinue(AdvisoryBoardDecisions.Deferred);
+			await _wizard.SetDecisionByAndContinue(DecisionMadeBy.RegionalDirectorForRegion);
+
+			Document.Url.Should().EndWith("/decision/why-deferred");
+		}
+
+		[Fact]
+		public void Should_display_the_selected_school_name()
+		{
+			var selectedSchool = Document.QuerySelector<IHtmlElement>("#selection-span").Text();
+
+			selectedSchool.Should().Be(_project.SchoolName);
+		}
+
+		[Fact]
+		public async Task Should_return_to_the_who_made_this_decision_page_when_back_link_is_clicked()
+		{
+			await NavigateAsync("Back");
+
+			Document.QuerySelector<IHtmlElement>("h1").Text().Should()
+				.Be("Who made this decision?");
```

Comment:
This will be broken by the page titles work, unfortunately - one to watch out for when they land.

Reply:
Looking forward to it 😁